.gitignore

```diff
@@ -1,2 +1,3 @@
-
+idea/
+*.iml
```

Comment:
I don't think that's needed... what's iml anyway?

Reply:
I used IntelliJ to edit this, but I've reverted it in 177a35aca8532581a8a5cd3d54102a7bf258ae80.